DifferenceGenerator/src/main/kotlin/algorithms/Gotoh.kt

```diff
@@ -167,6 +174,9 @@ class Gotoh<T>(
                         }
                     j--
                 }
+                else -> {
```

Comment:
My IDE marks this else branch as redundant and therefore unnecessary because the when is apparently "exhaustive".